packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -194,15 +202,5 @@ InteractionLayer.propTypes = {
   width: PropTypes.number.isRequired
 }
 
-InteractionLayer.defaultProps = {
```

Comment:
This isn't strictly required for this PR, but [`defaultProps` is being deprecated for functions](https://github.com/facebook/react/pull/16210), so it seemed like a good opportunity to remove it.